src/index.ts

```diff
@@ -88,6 +94,10 @@ export async function resolveSchema (config: Config): Promise<Schema> {
               })
           }
         })
+     case 'schema':
+       const schemaSource = require(resolve(config.file))
```

Comment:
"both of them" referring to the other file (schema.json) option? 

I think we should switch that one too. I'm not familiar with any API that takes a relative path that uses a leading slash. I think that's confusing.